nss-day-cohort-60/django-taters-api
# added create method to tags_view.py. Tested in postman and is working

Closes # 29


# Description of Proposed Changes

added create method to tags_view.py to allow POST request for client to create a new tag.

---


# Steps to Test

1. Fetch the `create-tags-MP` branch and switch to it
2. Start debugger and open postman
3. set method to POST http://localhost:8000/tags, include this in the post body:
{
        "label": "Boogers"
    }
and press send. Response body should be what was sent plus an id and 201 status code.
5. change to GET method and verify new tag is in the list. Verify it's in your database in VSCode, too.
6. proofread any changed modules (should just be tags_view.py)
